Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN dnf -y remove subscription-manager* && \
       qpid-proton-c-devel \
       ruby-devel \
       wget && \
+    mkdir -p /root/.gem/ruby # $GEM_HOME && \
```

Comment:
Should this live with the gem install bundler command?  Seems it has nothing to do with "dnf" per se.